test/create.js

```diff
@@ -118,11 +118,25 @@ test('no tampering with defaults', t => {
 		options: instance.defaults.options
 	});
 
-	// Tamper Time
-	t.throws(() => {
+	t.notThrows(() => {
 		instance.defaults.options.baseUrl = 'http://google.com';
 	});
 
-	t.is(instance.defaults.options.baseUrl, 'example');
+	t.is(instance.defaults.options.baseUrl, 'http://google.com');
 	t.is(instance2.defaults.options.baseUrl, 'example');
 });
+
+test('can prevent unwanted changes', t => {
+	const instance = got.create({
+		preventChanges: true,
```

Comment:
It's not clear what this does exactly? What changes does it prevent? All? I think it needs a more explicit name.